etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/database/PostgresDao.java

```diff
@@ -120,6 +123,7 @@ public synchronized PartnerMetadata fetchMetadata(String receivedSubmissionId)
 
             return new PartnerMetadata(
                     result.getString("message_id"),
+                    result.getString("sender"),
```

Comment:
Is this change going to be in the other PR that will add the sentSubmissionId handling?

Reply:
Yes